<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frm_agendamentoservicos</class>
 <widget class="QDialog" name="frm_agendamentoservicos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>646</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Agendamento de Serviços</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>8</y>
     <width>761</width>
     <height>631</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Novo Serviço</string>
    </attribute>
    <widget class="QPushButton" name="btn_agendarServico">
     <property name="geometry">
      <rect>
       <x>661</x>
       <y>568</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Agendar</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>72</y>
       <width>292</width>
       <height>27</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="txt_nomeCliente">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="placeholderText">
         <string>Nome Cliente</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txt_nomeVeiculo">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="placeholderText">
         <string>Modelo Veículo</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>52</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Dados cliente:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_selecionaCliente">
     <property name="geometry">
      <rect>
       <x>305</x>
       <y>73</y>
       <width>20</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>..</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>175</x>
       <y>112</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Hora</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTimeEdit" name="de_hora">
        <property name="calendarPopup">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_3">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>112</y>
       <width>150</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Data</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="de_data">
        <property name="dateTime">
         <datetime>
          <hour>0</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2022</year>
          <month>1</month>
          <day>1</day>
         </datetime>
        </property>
        <property name="maximumTime">
         <time>
          <hour>23</hour>
          <minute>59</minute>
          <second>59</second>
         </time>
        </property>
        <property name="currentSection">
         <enum>QDateTimeEdit::DaySection</enum>
        </property>
        <property name="displayFormat">
         <string>dd/MM/yyyy</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>10</y>
       <width>744</width>
       <height>271</height>
      </rect>
     </property>
     <property name="title">
      <string>Dados Serviço:</string>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>367</x>
        <y>117</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Observação</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_4">
      <property name="geometry">
       <rect>
        <x>363</x>
        <y>40</y>
        <width>181</width>
        <height>50</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Serviço</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_tipoServico"/>
       </item>
      </layout>
     </widget>
     <widget class="QPlainTextEdit" name="Ptxt_observacao">
      <property name="geometry">
       <rect>
        <x>364</x>
        <y>140</y>
        <width>221</width>
        <height>111</height>
       </rect>
      </property>
     </widget>
    </widget>
    <zorder>groupBox</zorder>
    <zorder>btn_agendarServico</zorder>
    <zorder>layoutWidget_2</zorder>
    <zorder>label_3</zorder>
    <zorder>btn_selecionaCliente</zorder>
    <zorder>layoutWidget</zorder>
    <zorder>layoutWidget_3</zorder>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Gestão Serviços</string>
    </attribute>
    <widget class="QTableWidget" name="tw_listaservicos">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>66</y>
       <width>736</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget_17">
     <property name="geometry">
      <rect>
       <x>416</x>
       <y>31</y>
       <width>331</width>
       <height>27</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLineEdit" name="txt_ge_filtrar"/>
      </item>
      <item>
       <widget class="QPushButton" name="btn_ge_filtrar">
        <property name="text">
         <string>Filtrar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_16">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>8</y>
       <width>141</width>
       <height>50</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Filtrar por:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cb_ge_filtrar"/>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>736</width>
       <height>261</height>
      </rect>
     </property>
     <property name="title">
      <string>Dados Serviço:</string>
     </property>
     <widget class="QWidget" name="layoutWidget_7">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>32</y>
        <width>181</width>
        <height>50</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Serviço</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_ge_tipoServico"/>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>103</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Observação</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="Ptxt_ge_observacao">
      <property name="geometry">
       <rect>
        <x>361</x>
        <y>129</y>
        <width>221</width>
        <height>111</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>169</y>
        <width>144</width>
        <height>50</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Status Serviço:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cb_ge_status"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_9">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>97</y>
        <width>150</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_12">
       <item>
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>Data</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="de_ge_data">
         <property name="dateTime">
          <datetime>
           <hour>0</hour>
           <minute>0</minute>
           <second>0</second>
           <year>2022</year>
           <month>1</month>
           <day>1</day>
          </datetime>
         </property>
         <property name="maximumTime">
          <time>
           <hour>23</hour>
           <minute>59</minute>
           <second>59</second>
          </time>
         </property>
         <property name="currentSection">
          <enum>QDateTimeEdit::DaySection</enum>
         </property>
         <property name="displayFormat">
          <string>dd/MM/yyyy</string>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_8">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>96</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_11">
       <item>
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Hora</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTimeEdit" name="de_ge_hora">
         <property name="calendarPopup">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>294</width>
        <height>52</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_13">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Dados cliente:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLineEdit" name="txt_ge_nomeCliente">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="placeholderText">
            <string>Nome Cliente</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txt_ge_nomeVeiculo">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="placeholderText">
            <string>Modelo Veículo</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="layoutWidget_10">
     <property name="geometry">
      <rect>
       <x>538</x>
       <y>566</y>
       <width>208</width>
       <height>27</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QPushButton" name="btn_ge_salvar">
        <property name="text">
         <string>Salvar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_ge_excluir">
        <property name="text">
         <string>Excluir</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
